modules/ztp-provisioning-lvm-storage.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-provisioning-lvm-storage_{context}"]
+= Configuring {lvms} using PolicyGenTemplate CRs
+
+You can configure {lvms-first} for managed clusters that you deploy with {ztp-first}.
+
+[NOTE]
+====
+You use {lvms} to persist event subscriptions when you use PTP events or bare-metal hardware events with HTTP transport.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Create a Git repository where you manage your custom site configuration data.
+
+.Procedure
+
+. To configure {lvms} for new managed clusters, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml,subs="attributes+"]
+----
+- fileName: StorageNS.yaml
```

Comment:
There are specific source-cr for LVMs. However, some overrides are needed since odf-lvm-operator was replaced by lvms-operator in 4.12:

```yaml
    - fileName: StorageLVMOSubscriptionNS.yaml
      policyName: subscription-policies
    - fileName: StorageLVMOSubscriptionOperGroup.yaml
      policyName: subscription-policies
    - fileName: StorageLVMOSubscription.yaml
      spec:
        name: lvms-operator
        channel: stable-4.12
        source: redhat-operator-index
      policyName: subscription-policies
```